charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 8.6.1
+version: 8.6.2
```

Comment:
new feature should raise minor version....

Reply:
sorry, bad bump version on my side